src/main/java/org/spongepowered/common/event/SpongeCauseStackManager.java

```diff
@@ -275,8 +277,8 @@ public void popCauseFrame(StackFrame oldFrame) {
     public static class CauseStackFrameImpl implements StackFrame {
 
         // lazy loaded
-        private Map<EventContextKey<?>, Object> stored_ctx_values;
-        private Set<EventContextKey<?>> new_ctx_values;
+        @Nullable private Map<EventContextKey<?>, Object> stored_ctx_values;
+        @Nullable private Set<EventContextKey<?>> new_ctx_values;
```

Comment:
Just curious, why are variables in snake case here?

Reply:
@Deamon5550 can tell you why

Reply:
Habit

Reply:
Well, this looks way to c++, which is in violation of Sponge code style. I think you can safely change them as they are not exposed in the API.

Reply:
I'm going to leave them as-is for now.